aldryn_news/views.py

```diff
@@ -31,6 +31,16 @@ class ArchiveView(BaseNewsView, ArchiveIndexView):
     template_name = 'aldryn_news/news_list.html'
     date_list_period = 'month'
 
+    @property
+    def uses_datetime_field(self):
+        """Return False.
```

Comment:
I don't know if this is specific to a version of HVAD or Django, but note that this will work fine in all versions, so I don't think it's necessary to do the check.